recipe/run_test.py

```diff
@@ -6,6 +6,9 @@
     from nose import run_exit
     from traits.etsconfig.api import ETSConfig
     import os
+    from subprocess import call
+    call("python setup.py build_ext --inplace",
```

Comment:
maybe os.path.join(filepath,'..','setup.py') should be constructed...
maybe call works from the path of file?